### ISSUE TITLE
wow. full apple platform support?

### ISSUE THREAD

**Issue by RichyZDenmark1332** [rryam/MusadoraKit#8]
Great library! But can it even support macOS, watchOS & tvOS?

 Do you plan to add support for these cross platform or only remain limited iOS?

**Comment by nevercode-rudrank** [comment]
Heya @RichyZDenmark1332! The package file has the support for the following platforms: 

```swift 
platforms: [.iOS(.v15), .macOS(.v12), .watchOS(.v8), .tvOS(.v15)]
```

Although, I have not tested it on the other platforms except iOS. For the new stuff, you see that I have annotated it with: 

```swift 
@available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *)
```

If you plan to use it, let me know if it works fine for you on other platforms. If not, you can raise issues here!

**Comment by rudrankriyam** [comment]
Heya @RichyZDenmark1332! The package file has the support for the following platforms:

```
platforms: [.iOS(.v15), .macOS(.v12), .watchOS(.v8), .tvOS(.v15)]
```

Although, I have not tested it on the other platforms except iOS. For the new stuff, you see that I have annotated it with:

```
@available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *)
```

If you plan to use it, let me know if it works fine for you on other platforms. If not, you can raise issues here!
